python/lsst/ts/cbp/mock_server.py

```diff
@@ -87,7 +107,7 @@ def __init__(self, log=None):
         self.park = False
         self.auto_park = False
         self.masks_rotation = {1: 0, 2: 0, 3: 0, 4: 0, 5: 0}
-        self.terminator = "\r\n"
+        terminator = b"\r\n"
```

Comment:
Please use `tcpip.DEFAULT_TERMINATOR` instead. Even better, remove this. See my next comment.